src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -1551,6 +1551,9 @@ public static void townToggle(CommandSender sender, String[] split, boolean admi
 					TownyMessaging.sendMsg(sender, String.format(TownySettings.getLangString("msg_changed_taxpercent"), town.isTaxPercentage() ? TownySettings.getLangString("enabled") : TownySettings.getLangString("disabled")));
 			} else if (split[0].equalsIgnoreCase("open")) {
 
+				if(TownySettings.isTownBankruptsyEnabled() && town.isBankrupt())
+					throw new TownyException(TownySettings.getLangString("msg_err_siege_besieged_town_cannot_toggle_open"));

```

Comment:
Language message should not be something relating to siegewar.

Reply:
Done